AUTHORS.md

```diff
@@ -9,4 +9,5 @@ Authors are sorted alphabetically.
 * Ken Odegard
```

Comment:
```suggestion
* John Kirkham
* Ken Odegard
```

Reply:
John has chosen not to use his full name in his github profile so that's why we didn't amend this. Is that a problem copyright-wise?

Reply:
I don't have a legal answer, I just know that we've sought to use the contributor's full name whenever possible, it helps when trying to deduplicate.